# Parameters for the CloudOdometer.

# TEASER++ robust registrator parameters.

# Square of ratio between acceptable noise and noise bound. Usually set to 1.
teaser_cbar2: 1.0
# When the solver does not estimate scale, it solves the registration problem
# much faster.
teaser_estimate_scaling: false
# A bound on the noise of each provided measurement.
teaser_noise_bound: 0.008; 
# Factor to multiple/divide the control parameter in the GNC algorithm.
teaser_rotation_gnc_factor: 1.4 
# Cost threshold for the GNC rotation estimators.
teaser_rotation_cost_threshold: 0.00001;


# 3D feature detection parameters
# For 2 meter scenario
norm_radius: 0.0375
fpfh_radius: 0.045

# Matching parameters
use_absolute_scale: true
use_crosscheck: true
use_tuple_test: false
tuple_scale: 0.975








